Source/SwiftyDropbox/Shared/Handwritten/DropboxClientsManager.swift

```diff
@@ -136,33 +141,34 @@ public class DropboxClientsManager {
         }
     }
 
-    public static func reauthorizeClient(_ tokenUid: String, sessionConfiguration: NetworkSessionConfiguration? = nil) {
+    public static func reauthorizeClient(_ tokenUid: String, sessionConfiguration: NetworkSessionConfiguration? = nil, transportClient: DropboxTransportClient? = nil) {
```

Comment:
nit: put args in the same order as setupAuthorizedClient

Reply:
ah, nice catch